wgpu-core/src/resource.rs

```diff
@@ -515,6 +515,8 @@ pub enum CreateTextureError {
     MultisampledNotRenderAttachment,
     #[error("Texture format {0:?} can't be used due to missing features.")]
     MissingFeatures(wgt::TextureFormat, #[source] MissingFeatures),
+    #[error("Sample count {0} is invalid")]
```

Comment:
Let's be a bit more specific here:

```suggestion
    #[error("Sample count {0} is not supported by format {1} on this device. It may be supported by your adapter through the TEXTURE_ADAPTER_SPECIFIC_FORMAT_FEATURES feature")]
```